### ISSUE TITLE
The PrintPreviewControl control have no dotted rectangle when using Keyboard focus on it

### ISSUE THREAD

**Issue by Zheng-Li01** [dotnet/winforms#3660]
* .NET Core Version:
.Net SDK 5.0.100-rc.1.20378.13

* Have you experienced this same bug with .NET Framework?:
Yes

**Minimal repro:**
1. Extract “[Keyboard_PreviewControl.zip](https://github.com/dotnet/winforms/files/4993308/Keyboard_PreviewControl.zip)” and open it in VS.
2. Build and run the project.
3. Navigating the controls on PrintingTesting Form through tab key.

**Expected behavior:**
Keyboard focus focus on PrintPreviewControl control, and the dot line can be seen after Preview Control button.

**Actual behavior:**
While navigating through tab key, Keyboard focus is not visible after Preview Control button.
![KeyBoard](https://user-images.githubusercontent.com/38325459/88771495-ea631500-d1b1-11ea-9ed5-6d8dcd187a86.gif)

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw We can quickly fix the test application to not focus on controls that can't be edited. However we should get guidance if a more comprehensive fix is needed - 
1. set default value for TabStop property to false
2. if the developer sets TabStop to true, we should draw the focus rectangle

**Comment by merriemcgaw** [comment]
@Zheng-Li01 please rebuild the accessibility sample app and retry this issue then. It should be fixed, though I see email that it still repros.

**Comment by Zheng-Li01** [comment]
@merriemcgaw, yes. the issue has been fixed after rebuild the accessibility sample app. see below GIF.
![CoreIssue](https://user-images.githubusercontent.com/38325459/89483319-5a832380-d7ce-11ea-8684-0f95abfd524b.gif)
